packages/react/src/components/Masthead/MastheadTopNav.js

```diff
@@ -126,6 +129,29 @@ function renderNav(link, autoid) {
   return navItems;
 }
 
+/**
+ * Checks if the current url exists in the list of links
+ *
+ * @param {object} obj The navigation list to search
+ * @param {string} target The URL to search for
+ * @returns {boolean} Whether or not the URL is found in the navigation list
+ */
+function hasCurrentUrl(obj, target) {
```

Comment:
Super minor thing, wondering if we should make this a private method?

Reply:
Oh yeah sure, good call.